### DISCUSSION TITLE
迅雷云盘无法下载、无法使用播放器播放视频

### DISCUSSION THREAD

**Discussion by wuyuzheyan** [AlistGo/alist#1149] [Invalid question]
迅雷云盘账号根据社区添加说明，正常添加，无报错，文件路径全部展示出来；
使用webdav挂载后
迅雷云盘无法下载、无法使用播放器播放视频

**Comment by czippo** [comment]
我也是

**Reply by thjjj** [reply]
我也是，我还以为是我的问题，阿里云盘正常，迅雷云盘不行，夸克云盘时好时坏，装的最新版

**Comment by NFolksa** [comment]
dddd，我也是

**Reply by sunzhe1234** [reply]
一样

**Comment by derbyoen** [comment]
解决了吗

**Reply by wakaka123wakaka** [reply]
我倒是可以播放，但是私密空间文件夹不能播放

**Comment by wakaka123wakaka** [comment]
播放私密文件夹的内容提示这个
{"code":500,"message":"failed get link: ErrorCode: 7 ,Error: file_moved_to_safe ,ErrorDescription: 该文件已经移入超级保险箱，请升级到最新版本 ","data":null}

**Reply by earor-R** [reply]
我的百度网盘也是这个情况

**Reply by wakaka123wakaka** [reply]
百度网盘私密空间还是普通的文件夹呢

**Reply by earor-R** [reply]
> 百度网盘私密空间还是普通的文件夹呢

普通的，我试了两个号都是这样

**Comment by kuko7** [comment]
请问你现在解决了吗？我添加了迅雷云盘，一切都正常，但是使用Infuse点开播放几秒或者十几秒就卡住了，也不能下载，不知道怎么解决。
